aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -54,37 +54,15 @@ The following services are registered during host initialization:
 
 ## Host configuration
 
-[HostBuilder](/dotnet/api/microsoft.extensions.hosting.hostbuilder) relies on the following approaches to set the host configuration values:
+Host configuration is created by:
 
-* Configuration builder
-* Extension method configuration
+* Setting a property for the [application key (name)](#application-key-name) in [ConfigureAppConfiguration](#configureappconfiguration).
```

Comment:
This is weird. I don't think the app name is used for anything yet so I'd rather exclude the code sample from docs until 3.0.